### PR TITLE
Revise the expressions that can be pushed down

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Update the expressions and data types that TiDB supports to push down.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
   ① https://github.com/pingcap/docs-cn/pull/8684
   ② https://github.com/pingcap/docs-cn/pull/8761 (minor updates)
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/expressions-pushed-down.md [hunk 1]
```diff
@@ -39,29 +39,32 @@ tidb> desc mysql.expr_pushdown_blacklist;
...
++ `name`: the name of the function, operator, or data type that is prohibited from being pushed down.
++ `store_type`: specifies to which storage engine the function, operator, or data type is prohibited from being pushed down. Currently, TiDB supports the three storage engines: `tikv`, `tidb`, and `tiflash`. `store_type` is case-insensitive. If a function is prohibited from being pushed down to multiple storage engines, use a comma to separate each engine.
 + `reason` : The reason why the function is blocklisted.
 
 ### Add to the blocklist
 
-To add one or more functions or operators to the blocklist, perform the following steps:
+To add one or more [functions, operators](#supported-expressions-for-pushdown), or data types (**only** [`ENUM` type](/data-type-string.md#enum-type) and [`BIT` type](/data-type-numeric.md#bit-type)) to the blocklist, perform the following steps:
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:48]:
```suggestion
To add one or more [functions, operators](#supported-expressions-for-pushdown), or data types (**only** the [`ENUM` type](/data-type-string.md#enum-type) and the [`BIT` type](/data-type-numeric.md#bit-type)) to the blocklist, perform the following steps:
```

File: system-variables.md [hunk 2]
```diff
@@ -1632,6 +1632,12 @@ SET tidb_slow_log_threshold = 200;
 - Range: `[0, 9223372036854775807]`
 - This variable is used to limit the maximum number of requests TiDB can send to TiKV at the same time. 0 means no limit.
 
+### tidb_sysdate_is_now （New in v6.0.0）
+
+- Scope: SESSION | GLOBAL
+- Default value: `OFF`
+- This variable is used to control whether `SYSDATE` can be replaced by `NOW`. This configuration item has the same effect as the MySQL option [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now).
```
Comment (TomShawn) [inline system-variables.md:1639]:
```suggestion
- This variable is used to control whether the `SYSDATE` function can be replaced by the `NOW` function. This configuration item has the same effect as the MySQL option [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now).
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

File: functions-and-operators/expressions-pushed-down.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ When TiDB reads data from TiKV, TiDB tries to push down some expressions (includ
 
 ## Blocklist specific expressions
 
-If unexpected behavior occurs during the calculation of a function caused by its pushdown, you can quickly restore the application by blocklisting that function. Specifically, you can prohibit an expression from being pushed down by adding the corresponding functions or operator to the blocklist `mysql.expr_pushdown_blacklist`.
+Suppose that unexpected behavior occurs in the calculation process when pushing down the [supported expressions](#supported-expressions-for-pushdown) or specific data types (**only** the [`ENUM` type](/data-type-string.md#enum-type) and the [`BIT` type](/data-type-numeric.md#bit-type)). In this case, you can restore the application quickly by prohibiting the pushdown of the corresponding functions, operators, or data types. Specifically, you can prohibit the functions, operators, or data types from being pushed down by adding them to the blocklist `mysql.expr_pushdown_blacklist`. For details, refer to [Add to the blocklist](#add-to-the-blocklist).
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:24]:
```suggestion
If unexpected behavior occurs in the calculation process when pushing down the [supported expressions](#supported-expressions-for-pushdown) or specific data types (**only** the [`ENUM` type](/data-type-string.md#enum-type) and the [`BIT` type](/data-type-numeric.md#bit-type)), you can restore the application quickly by prohibiting the pushdown of the corresponding functions, operators, or data types. Specifically, you can prohibit the functions, operators, or data types from being pushed down by adding them to the blocklist `mysql.expr_pushdown_blacklist`. For details, refer to [Add to the blocklist](#add-to-the-blocklist).
```

**Comment by TomShawn** [comment]
LGTM

**Review by Yisaer — APPROVED** [review]

**Pull request merged**